### ISSUE TITLE
Evento sai só ou alguém precisa remover

### ISSUE THREAD

**Issue by williamespindola** [saiadecasa/saiadecasa.github.io#379]
Se a idéia de novos métodos de colaboração #377 vingarem seria uma boa os antigos não serem listados e serem removidos durante algum build de publicação de novos.

**Comment by khullah** [comment]
Nao pode manter eventos passados no json e só não mostrar na página?

**Comment by airtonzanon** [comment]
Isso deixaria o arquivo muito sujo, não seria algo muito bom.

Porém a ideia de remover do arquivo em algum build é bem boa @williamespindola

**Comment by rafa-acioly** [comment]
concordo com o @khullah, listar apenas os vigentes, com relação a remover do arquivo; acredito que os dados antigos poderiam ser usados para analisar o cenário como um todo por exemplo;

1. qual o período do ano existem mais eventos?
2. em quais cidades existem mais eventos sobre "X"?
3. qual a média de duração de um evento "Y"?


Todos esses dados seriam uteis para os organizadores e curiosos.

**Comment by airtonzanon** [comment]
Isso é algo bem interessante para fazermos @rafa-acioly, gostei da ideia.

Talvez, consigamos pegar os dados antigos vasculhando o git.

**Comment by rafa-acioly** [comment]
se a ideia vingar eu me disponho a ajudar para criar uma pagina com os dados 🎉

**Comment by airtonzanon** [comment]
Vamos ver o que a galera acha disso e acho que podemos fazer sim. Só precisamos pensar na forma de implementar isso.

cc/ @williamespindola @rogeriopradoj
